libsolidity/formal/SMTEncoder.cpp

```diff
@@ -2688,7 +2674,29 @@ vector<VariableDeclaration const*> SMTEncoder::stateVariablesIncludingInheritedA
 
 vector<VariableDeclaration const*> SMTEncoder::localVariablesIncludingModifiers(FunctionDefinition const& _function, ContractDefinition const* _contract)
 {
-	return _function.localVariables() + modifiersVariables(_function, _contract);
+	return _function.localVariables() + tryCatchVariables(_function) + modifiersVariables(_function, _contract);
+}
+
+vector<VariableDeclaration const*> SMTEncoder::tryCatchVariables(FunctionDefinition const& _function)
+{
+	struct TryCatchVarsVisitor : public ASTConstVisitor
+	{
+		bool visit(TryCatchClause const& _catchClause) override
+		{
+			if (_catchClause.parameters())
+			{
+				auto const& params = _catchClause.parameters()->parameters();
+				for (auto param: params)
+					m_vars.push_back(param.get());
+			}
+
+			return false;
+		}
+
+		vector<VariableDeclaration const*> m_vars;
```

Comment:
I still think this should be called `vars` :p
We usually add `m_` for class private members only. I know you said this ends up being private, but the way it's accessed is still public (`tryCatchVarsVisitor.m_vars`).

Reply:
Fixed.